src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -352,6 +347,13 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 End If
             Next
         End Sub
+
+        Private Sub LnkLabel_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles lnkLabel.LinkClicked
+            TelemetryLogger.LogAdvBuildSettingsPropPageEvent(TelemetryLogger.AdvBuildSettingsPropPageEvent.LangVersionMoreInfoLinkClicked)
+
+            Dim url = DirectCast(e.Link.LinkData, String)
+            Process.Start(url)
```

Comment:
This can throw a Win32Exception in certain circumstances, such as if the default URL handler isn't installed.

Reply:
Noted. I'll add a try/catch.